Mathlib/Analysis/Meromorphic/Order.lean

```diff
@@ -87,6 +87,34 @@ lemma order_eq_int_iff {n : ℤ} (hf : MeromorphicAt f x) : hf.order = n ↔
     exact ⟨fun h ↦ ⟨g, hg_an, hg_ne, h ▸ hg_eq⟩,
       AnalyticAt.unique_eventuallyEq_zpow_smul_nonzero ⟨g, hg_an, hg_ne, hg_eq⟩⟩
 
+lemma order_neg_iff_tendsto_cobounded (hf : MeromorphicAt f x) :
+    hf.order < 0 ↔ Tendsto f (𝓝[≠] x) (Bornology.cobounded E) := by
+  rcases lt_or_le hf.order 0 with h'f | h'f
+  · simp only [h'f, ← tendsto_norm_atTop_iff_cobounded, true_iff]
+    have : hf.order ≠ ⊤ := h'f.ne_top
+    obtain ⟨m, ho⟩ := WithTop.ne_top_iff_exists.mp this
+    have m_neg : m < 0 := by simpa [← ho] using h'f
+    rcases hf.order_eq_int_iff.1 ho.symm with ⟨g, g_an, gx, hg⟩
+    have A : Tendsto (fun z ↦ ‖(z - x) ^ m • g z‖) (𝓝[≠] x) atTop := by
+      simp only [norm_smul]
+      apply Filter.Tendsto.atTop_mul_pos (C := ‖g x‖) (by simp [gx]) _
+        g_an.continuousAt.continuousWithinAt.tendsto.norm
+      have : Tendsto (fun z ↦ (z - x)) (𝓝[≠] x) (𝓝[≠] 0) := by
+        refine tendsto_nhdsWithin_iff.2 ⟨?_, ?_⟩
+        · have : ContinuousWithinAt (fun z ↦ z - x) ({x}ᶜ) x :=
+            ContinuousAt.continuousWithinAt (by fun_prop)
+          simpa using this.tendsto
+        · filter_upwards [self_mem_nhdsWithin] with y hy
+          simpa [sub_eq_zero] using hy
+      apply Tendsto.comp (NormedField.tendsto_norm_zpow_nhdsNE_zero_atTop m_neg) this
+    apply A.congr'
+    filter_upwards [hg] with z hz using by simp [hz]
+  · simp only [lt_iff_not_ge, ge_iff_le, h'f, not_true_eq_false, false_iff]
+
+
+
+#exit
```

Comment:
I am unsure what to make of this `#exit`. Is this on purpose, or did the line enter by mistake?

Reply:
The PR is not in a ready to review state, sorry!

Reply:
The PR was ready when I asked for your review yesterday, but then Michael made some very good comments that I could add some lemmas to make it better, so I am working on the changes right now, and the PR is currently a mess. When it's clean again (hopefully in a few hours), I'll remove the `awaiting-author` tag.

Reply:
Roger. I will wait for the removal of `awaiting-author` then. See you in a while!

Reply:
The PR is ready. Thanks for waiting!